gnovm/pkg/gnolang/machine.go

```diff
@@ -1139,6 +1139,10 @@ const (
 	OpCPURangeIterMap      = 48
 	OpCPURangeIterArrayPtr = 46
 	OpCPUReturnCallDefers  = 78
+
+	/* Print */
+	OpCPUPrintln = 1
+	OpCPUPrint   = 1
```

Comment:
Can you get this information from benchops, rather than just assuming 1?

Reply:
Do you know how to calculate the related gas with this ?

Reply:
like benchmark other OpCodes to calculate this.
however i'm good with a "guess value" here within this PR. Since there are some other places with the same issue ie GC. 
Eventually, we need a comprehensive  benchmark after to fulfill all of these.